libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/components/MatrixUserHeader.kt

```diff
@@ -41,44 +37,65 @@ import io.element.android.libraries.designsystem.theme.components.Text
 import io.element.android.libraries.matrix.api.user.MatrixUser
 import io.element.android.libraries.matrix.ui.model.getAvatarData
 import io.element.android.libraries.matrix.ui.model.getBestName
+import io.element.android.libraries.theme.ElementTheme
 
 @Composable
 fun MatrixUserHeader(
+    matrixUser: MatrixUser?,
+    modifier: Modifier = Modifier,
+    // onClick: () -> Unit = {},
+) {
+    if (matrixUser == null) {
+        MatrixUserHeaderPlaceholder(modifier = modifier)
+    } else {
+        MatrixUserHeaderContent(
+            matrixUser = matrixUser,
+            modifier = modifier,
+            // onClick = onClick
+        )
+    }
+}
+
+@Composable
+private fun MatrixUserHeaderContent(
     matrixUser: MatrixUser,
     modifier: Modifier = Modifier,
-    onClick: () -> Unit = {},
+    // onClick: () -> Unit = {},
```

Comment:
Is this (and the other) comment a leftover or is it intended?

Reply:
Actually at some point we will have a onClick action here, but by keeping the `onClick` on the modifier, we have the ripple effect, but no action which looks like a bug. So I commented out that, then the param appears as unused, so I commented it out also.

I have added a TODO to explain why I do not really want to remove this code.

Reply:
c8afa7ef2e42ae842dd8435cb12421c58543c22f